bundles/ch.elexis.core.application/plugin_it.properties

```diff
@@ -1,27 +1,27 @@
 
 elexis.AccountListView            = Lista account
-elexis.AccountView                = Conto
-elexis.BillSummaryView            = Fattura panoramica
+elexis.AccountView                = Account
```

Comment:
Why do you use here an english word?
The other  translation look okay for me (but I only learned 7 years of latin) and nothing of italian at school.